src/transformers/models/clip/modeling_clip.py

```diff
@@ -660,7 +660,9 @@ def forward(
 
         # text_embeds.shape = [batch_size, sequence_length, transformer.width]
         # take features from the eot embedding (eot_token is the highest number in each sequence)
-        pooled_output = last_hidden_state[torch.arange(last_hidden_state.shape[0]), input_ids.argmax(dim=-1)]
+        pooled_output = last_hidden_state[
+            torch.arange(last_hidden_state.shape[0]), input_ids.to(torch.int).argmax(dim=-1)
```

Comment:
I would add a comment here to say that the cast `to(torch.int)` is required for ONNX export.

Reply:
sure, added a comment and pushed

Reply:
Actually ONNX does support `int64`. But from what I read [here](https://github.com/microsoft/onnxruntime/issues/10068), ArgMax does not support `int64` inputs with opset 14. So I would just put something like that:
```
# casting to torch.int for onnx compatibility: argmax doesn't support int64 inputs with opset 14
```

Reply:
ah, my bad. updated the comment and pushed